source/beamline/BioXAS/BioXASFrontEndShutters.cpp

```diff
@@ -12,13 +12,12 @@ BioXASFrontEndShutters:: BioXASFrontEndShutters(const QString &name, QObject *pa
 
 	upstreamPhotonShutter_ = 0;
 	downstreamPhotonShutter_ = 0;
-	safetyShutter_ = 0;
 
 	// Current settings.
 
 	setUpstreamPhotonShutter(new AMReadOnlyPVControl("IPSH1407-I00-01", "IPSH1407-I00-01:state", this));
 	setDownstreamPhotonShutter(new CLSExclusiveStatesControl("IPSH1407-I00-02", "IPSH1407-I00-02:state", "IPSH1407-I00-02:opr:open", "IPSH1407-I00-02:opr:close", this));
-	setSafetyShutter(new CLSExclusiveStatesControl("SSH1407-I00-01", "SSH1407-I00-01:state", "SSH1407-I00-01:opr:open", "SSH1407-I00-01:opr:close", this));
```

Comment:
Do we manage the safety shutter elsewhere, instead of with the other front-end shutters?